css/gallery.css

```diff
@@ -51,7 +51,156 @@
 
 .gallery__items :nth-child(2n) {
```

Comment:
Can you explain to me what it is ":nth-child(2n) "

Reply:
that class is not mine is form frank. and that is all ready in develop